src/main/webapp/components/workspace/workspace.js

```diff
@@ -236,6 +250,13 @@ export default () => {
                   })
                 }}
               />
+              <Filters
```

Comment:
I think it would be better have the filters component keep track of the enabled/disabled state instead

Reply:
I tried that approach first but we need to disable the filters when not enabled in the workspace component. How would you go about pushing it down?

Reply:
In that case, you would still use the onChange to set filters in the workspace to [] , while keeping the filters stored on the Filters component.